contrib/ansible/roles/node/tasks/main.yml

```diff
@@ -3,11 +3,10 @@
   command: getenforce
   register: selinux
   changed_when: false
-  when: ansible_selinux
 
 - name: Set selinux permissive because tokens and selinux don't work together
   selinux: state=permissive policy=targeted
-  when: ansible_selinux and 'Enforcing' in selinux.stdout
+  when: "'Enforcing' in selinux.stdout"
```

Comment:
If I understand you correctly, I removed the getenforce section completely and changed the "Set selinux.... " section to following:

- name: Set selinux permissive because tokens and selinux don't work together
  selinux: state=permissive policy=targeted
  when: ansible_selinux

Thats what you mean? But I dont see ansible_selinux defined anywhere, is that a keyword?